lib/superstore/associations/reflection.rb

```diff
@@ -55,6 +55,9 @@ def inverse_name
         options[:inverse_of]
       end
 
+      def parent_reflection
+      end
```

Comment:
[Here's](https://github.com/rails/rails/blob/7883100e0f1e6a31b2bbd6377eef9272abb792ca/activerecord/lib/active_record/reflection.rb#L80) the line that makes this necessary. I'm not sure what it's used for, but it seems to be `nil` for every association type I can find.

Reply:
It's used for [HABTM associations](https://github.com/rails/rails/blob/7883100e0f1e6a31b2bbd6377eef9272abb792ca/activerecord/lib/active_record/associations.rb#L1977)